authorized_keys

```diff
@@ -85,3 +85,5 @@ ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDAzg8e0cW4io7PVp+ID/f8wHldye9eMXv3wwp1TRpo
 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCqb4RqdSdDPM6A20l2aXjG4v9K99ogsr8lpBGHHl7V2Ek00mrS3bS4sCbvp3YfUFNMZfz+8Fbbl/KIJJngISAw4QTcLMPoUSVfm0H23neJ+LeJjfC6WFlF1orXuTothBjhXvGmEUnpdjfMdA+qNDGNRaVxKBrlCQR1j3JtBzPKBSk/4tAKotOfNmiQlzWPCU7Ea7f96m4I5kimgnuUcFQvhSR+t37JYMeTPu/e3j/nuzVf8dk0yeCHtVvw9CiT7lhho1B/9bM999xDR4scOSBWna2rLrFECn7Ao2d/PoQ5wZE2DRyfXRtonH053VE15Jpc+LJI7VOWXnFONWym8Bu7
 
 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDmUqhU3Eg6zPJ8MEKpCcBkYRHXd/FVBxmy0DOj3GD5Hdxd/+cBDk/mfIJOy+r4HG2qQJBls9EvvZzcSnF3mqqURfWUDfKJ+fNZ+0vdeTodmgHiOERO5pUyf4h1YxqbxX+cVfELxpPwkGjtWIsvdt+CKuQ1b1rauKgv33YVkm1VJHCBJk8mh/JN5jsplnyu0nR5sPOJQ8rnDOr3vRGa0nGc+G7S03bAUBTmgjbEsRUOG2TJ1TPmkXoCVlR1I4yWUmgSMiKAO8nOFEwQaOQbLhLsvcYF7T4KzF86BUIG6hkm3qfHLPCmdbOfXfFnp8oNZiBq/S0nrHMLT9BGXjfXdbGv sm101
+
+ssh-dss AAAAB3NzaC1kc3MAAACBAMdlCk1NNT2O+np4uzFWFDHP+zTS8uAC6c0mv2miSgAgJxFyfZpJH+HbOuLALCoyPrQbAPb+yPeXvl7xQwAUd94QW3dsX8B70skaxGQMXJdvEu3iDSnpxdeNMW+Ctl4JDHwNoZ93dCxqUqiF5tIE9ock8r1vEZ4d4Xy/LWe+mneVAAAAFQCZ3YEG7uDAfKRxcIK7v4XJyCknCwAAAIA4l8xAexLrEiheg8w8YYGvTtTV20xDaFObLI0fWFpYM0n6g80xkGoM409/1ne6PPqOydCp6dfNcbqf2vCq2WxffjEetMSE5BNk02JctdafO8wiGVFnQd39I+n70SCU/48s/NX+RqWcRgTlwDzp034ZiclDrmrBGVmz5TAJWXT8BgAAAIBbYv/+kxyNdM0HLiQn6/ShTCqK6gkhumDn3a/SS0nHx3LpdlACX9x49a7VTf4tYqctW6LUkE9ei0cvsHWq2ec6Q00UAypCaTtwUjt7vr7HmwuTKV6XOsLkupnEED5jtRgeEz5fuWPIMH6Xg/GENJ5z7N/6AlaOz3Emu6TQtkdwPw== mwiget@mwiget-mba13
```

Comment:
Oh I suppose you don't need a rebase: in this case you can simply `merge` up to the commits that you want, since the ones that you don't want come later.

(Sorry for rapid-fire comments... :-))